pages/_app.tsx

```diff
@@ -18,6 +18,14 @@ const unauthenticatedRoutes = [
   '/oauth/jwks',
   '/idp/select',
   '/error',
+  '/setup/[token]',
```

Comment:
@ukrocks007 @devkiran We should rethink this way of filtering unauthenticated routes, might be good to make it blog based. For ex. `/setup/*`

Reply:
Yes, already noticed. I'll make the changes.